samples/meetings-live-code-interview/nodejs/ClientApp/package.json

```diff
@@ -16,8 +17,8 @@
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.21.4",
     "cors": "^2.8.5",
-    "has-cors": "^1.1.0",
     "fluid-framework": "~0.59.0",
+    "has-cors": "^1.1.0",
```

Comment:
What is the use of this package?

Reply:
Providing a Connect/Express middleware that can be used to enable.